### ISSUE TITLE
defaults.properties is not currently provided by any debian package

### ISSUE THREAD

**Issue by pmauduit** [georchestra/datadir#167]
As far as I know, no debian package is configured to claim the ownership of the `/etc/georchestra/default.properties` file, which means that a default setup based on these packages might be broken (or we have a good luck with default parameter values).

Maybe the Security-proxy should take care of the file ?

**Comment by landryb** [comment]
totally agree, saw that and i didnt really know what to do with this issue... but generally, i'd say a good practice is to checkout a datadir from git to keep track of files (and eventually compare with the `.dpkg-dist` files upon pkg upgrades). So i'm not sure if its a real issue (it is one for ppl who dont checkout a datadir)

**Comment by pmauduit** [comment]
My concern is in case we want to split the infrastructure onto several different servers, why would one want to force installing the SP if the machine is supposed to be dedicated to GN or GS ?

**Comment by landryb** [comment]
Good point - then i see no other way than having a georchestra-config pkg shipping only this file, and making it mandatory. ugly.

**Comment by pmauduit** [comment]
we could also build a package "georchestra-common" from this repository, and having by default as a dependency from our packages ? I shall see how to create a debian package (outside of maven I mean)
